sdk/identity/azure-identity/samples/azure-aad-auth-with-redis-py.md

```diff
@@ -0,0 +1,107 @@
+## Azure Cache for Redis: Azure AD with redis client library
+
+### Table of contents
+
```

Comment:
Add Pre-req section with following content as idiomatically applicable:

- Configuration of Role and Role Assignments is required before using the sample code in this document.
- **Dependency Requirements:**
   ```xml
 
   ```